src/universal/modules/teamDashboard/components/AgendaItem/AgendaItem.js

```diff
@@ -243,46 +123,40 @@ class AgendaItem extends Component {
     const canDelete = !isComplete && !isCurrent && !disabled;
     const inAgendaGroupLocal = inAgendaGroup(localPhase);
     const inAgendaGroupFacilitator = inAgendaGroup(facilitatorPhase);
-
-    // Root block styles
-    const rootStyles = agendaTopicRootStyles;
-    const addRootStyles = css(
-      rootStyles.rootBlock,
-      inAgendaGroupLocal && isLocal && rootStyles.rootActiveLocal,
-      inAgendaGroupFacilitator && isFacilitator && !inSync && rootStyles.rootOutOfSync,
-      isComplete && rootStyles.rootIsComplete,
-      disabled && rootStyles.rootDisabled,
-      isComplete && disabled && rootStyles.rootIsCompleteDisabled
-    );
-
-    // Content link styles
-    const linkStyles = agendaTopicLinkStyles;
-    const addLinkStyles = css(
-      linkStyles.linkBase,
-      isComplete && linkStyles.linkIsComplete,
-      canNavigate && linkStyles.linkCanNavigate,
-      inAgendaGroupLocal && isLocal && linkStyles.linkActiveLocal,
-      inAgendaGroupFacilitator && isFacilitator && !inSync && linkStyles.linkOutOfSync
+    const isOutOfSync = inAgendaGroupFacilitator && isFacilitator && !inSync;
+    const navItemState = {
```

Comment:
@mattkrick I was able to boil this file down and add the state for active, complete, out of sync etc. to the new `MeetingSubnavItem` and I need similar in the discuss topic nav component.